kitchen.ec2.yml

```diff
@@ -31,6 +31,7 @@ driver:
     instance_metadata_tags: 'enabled'
   #iam_profile_name: chef-client
   instance_type: <%= ENV['KITCHEN_INSTANCE_TYPE'] || "t2.micro" %>
+  iam_profile_name: <%= ENV['AWS_IAM_PROFILE'] %>
```

Comment:
1. You should document what profile is required to run the test
2. This breaks the possibility to run tests without setting env variables, relying on defaults. Could we think of a default? Is there a way to leverage tags?